.ansible-lint

```diff
@@ -1,11 +1,4 @@
 ---
 skip_list:
-- '106'
-- '204'
-- '208'
 - '301'  # Commands should not change things if nothing needs doing
 - '303'  # systemctl used in place of systemd module
```

Comment:
I'm assuming this will remain - probably tests that use `systemctl` which is ok

Reply:
Yeah, `systemd` module does not support `systemctl status` to get the full status message...